internal/pkg/config/config.go

```diff
@@ -32,56 +32,56 @@ type Config struct {
-	HQRateLimitingSendBack bool     `mapstructure:"hq-rate-limiting-send-back"`
-	NoBatchWriteWAL        bool     `mapstructure:"ultrasafe-queue"`
-	Handover               bool     `mapstructure:"handover"`
+	UserAgent                    string   `mapstructure:"user-agent"`
+	Cookies                      string   `mapstructure:"cookies"`
+	APIPort                      string   `mapstructure:"api-port"`
+	PrometheusPrefix             string   `mapstructure:"prometheus-prefix"`
+	WARCPrefix                   string   `mapstructure:"warc-prefix"`
+	WARCOperator                 string   `mapstructure:"warc-operator"`
+	WARCTempDir                  string   `mapstructure:"warc-temp-dir"`
+	WARCSize                     int      `mapstructure:"warc-size"`
+	WARCOnDisk                   bool     `mapstructure:"warc-on-disk"`
+	WARCPoolSize                 int      `mapstructure:"warc-pool-size"`
+	WARCDedupeSize               int      `mapstructure:"warc-dedupe-size"`
+	CDXDedupeServer              string   `mapstructure:"warc-cdx-dedupe-server"`
+	CDXCookie                    string   `mapstructure:"warc-cdx-cookie"`
+	HQAddress                    string   `mapstructure:"hq-address"`
+	HQKey                        string   `mapstructure:"hq-key"`
+	HQSecret                     string   `mapstructure:"hq-secret"`
+	HQProject                    string   `mapstructure:"hq-project"`
+	HQStrategy                   string   `mapstructure:"hq-strategy"`
+	HQBatchSize                  int      `mapstructure:"hq-batch-size"`
+	HQBatchConcurrency           int      `mapstructure:"hq-batch-concurrency"`
+	DisableHTMLTag               []string `mapstructure:"disable-html-tag"`
+	ExcludeHosts                 []string `mapstructure:"exclude-host"`
+	IncludeHosts                 []string `mapstructure:"include-host"`
+	IncludeString                []string `mapstructure:"include-string"`
+	ExcludeString                []string `mapstructure:"exclude-string"`
+	ExclusionFile                []string `mapstructure:"exclusion-file"`
+	WorkersCount                 int      `mapstructure:"workers"`
+	MaxConcurrentAssetsPerWorker int      `mapstructure:"max-concurrent-assets-per-worker"`
+	MaxHops                      int      `mapstructure:"max-hops"`
+	MaxRedirect                  int      `mapstructure:"max-redirect"`
+	MaxRetry                     int      `mapstructure:"max-retry"`
+	HTTPTimeout                  int      `mapstructure:"http-timeout"`
+	HTTPReadDeadline             int      `mapstructure:"http-read-deadline"`
+	CrawlTimeLimit               int      `mapstructure:"crawl-time-limit"`
+	CrawlMaxTimeLimit            int      `mapstructure:"crawl-max-time-limit"`
+	MinSpaceRequired             int      `mapstructure:"min-space-required"`
+	KeepCookies                  bool     `mapstructure:"keep-cookies"`
+	Headless                     bool     `mapstructure:"headless"`
```

Comment:
Should be removed

Reply:
Nothing to do with archiver, let's not stuff this PR with unrelated (but overall relevant) stuff

Reply:
Let's make an issue to purge (or re-implement) config entries instead of stuffing that in a PR that treats concurrency